src/main/java/uk/nhs/tis/trainee/notifications/model/NotificationType.java

```diff
@@ -44,6 +44,7 @@ public enum NotificationType {
   LTFT("less-than-full-time"),
   NON_EMPLOYMENT("non-employment"),
   PLACEMENT_INFORMATION("placement-information"),
+  PLACEMENT_USEFUL_INFORMATION("placement-useful_information"),
```

Comment:
Is this meant to be `placement-useful-information` instead of `placement-useful_information`?